com.microsoft.mrtk.accessibility/Subsystems/IAccessibilitySubsystem.cs

```diff
@@ -2,19 +2,45 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Collections.Generic;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.Accessibility
 {
     /// <summary>
-    /// Cross-platform, portable set of specifications for what
-    /// an Accessibility Subsystem is capable of. Both the Accessibility
-    /// subsystem and the associated provider must implement this interface,
-    /// preferably with a direct mapping or wrapping between the provider
-    /// surface and the subsystem surface.
+    /// Cross-platform, portable set of specifications for which an Accessibility Subsystem is capable. Both the Accessibility
+    /// subsystem and the associated provider must implement this interface, preferably with a direct mapping or wrapping between
+    /// the provider surface and the subsystem surface.
     /// </summary>
     public interface IAccessibilitySubsystem
     {
+        #region Describable object management
```

Comment:
(could be reserved for a future iteration, but) do you think it would be good to split these types of accessibility subsystems into their own interfaces? Like `IDescribableAccessibilitySubsystem` and `ITextColorAccessibilitySubsystem` or something

Reply:
let's revisit this thought when the accessibility feature set grows.